bundle/cert-manager-operator/0.15.2/cert-manager-operator.v0.15.2.clusterserviceversion.yaml

```diff
@@ -0,0 +1,89 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    description: "x509 certificate management for Kubernetes"
+    createdAt: "2020-05-04T12:00:00Z"
+    support: "cert-manager team"
+    alm-examples: ' [{"apiVersion":"operator.cert-manager.io/v1alpha1","kind":"CertManager","metadata":{"name":"cert-manager","namespace":"cert-manager"},"spec":{}}]'
```

Comment:
not sure if this effects others, but do we want a whitespace after `'`?

Reply:
oops...